keyboards/binepad/bn003/rules.mk

```
@@ -0,0 +1,33 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
+BOOTLOADER = atmel-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = no      # Virtual DIP switch configuration
```

```suggestion
BOOTMAGIC_ENABLE = lite      # Virtual DIP switch configuration
```
I heavily recommend having at least `lite` enabled and configured, especially when the board does not have an easy to access reset switch.